### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by wleoncio** [ocbe-uio/RCFF#9]
- [ ] r2cff
- [ ] cff2r (depends on #3 being closed)

**Comment by wleoncio** [comment]
Download DESCRIPTION file from several (popular?) R packages to use as test units.

**Comment by wleoncio** [comment]
Project was superseded, won't fix.
